tests/queries/0_stateless/00804_test_alter_compression_codecs.sql

```diff
@@ -54,15 +62,17 @@ ALTER TABLE alter_bad_codec ADD COLUMN alter_column DateTime DEFAULT '2019-01-01
 
 ALTER TABLE alter_bad_codec ADD COLUMN alter_column DateTime DEFAULT '2019-01-01 00:00:00' CODEC(ZSTD(100)); -- { serverError 433 }
 
+ALTER TABLE alter_bad_codec ADD COLUMN alter_column DateTime DEFAULT '2019-01-01 00:00:00' CODEC(DEFLATE_QPL(100)); -- { serverError 378 }
```

Comment:
Instead of a cryptic error number, it is possible to write out the symbolic error name (as in Common/ErrorCodes.cpp), e.g. `-- { DATA_TYPE_CANNOT_HAVE_ARGUMENTS }`.

Reply:
Done